lte/gateway/python/magma/pipelined/app/enforcement_stats.py

```diff
@@ -409,8 +409,8 @@ def _report_usage(self, usage):
         record_table = RuleRecordTable(records=usage.values(),
                                        epoch=global_epoch)
         if self._print_grpc_payload:
-            record_msg = 'Sending RPC payload: {0}{{\n{1}}}'.format(
-                record_table.DESCRIPTOR.name, str(record_table))
+            record_msg = 'Sending RPC payload: \n{0}{{{1}}}'.format(
+                record_table.DESCRIPTOR.name, "\n"+str(record_table))
```

Comment:
not sure if we are trying to ident the whole message or just to separate headers here. But note this will only add \n to the first line. The rest of the lines will not have any indentation

Reply:
It was just meant to separate headers.

Reply:
@veshkemburu can you show the difference between before/after this change?